src/BlazorUI/Bit.BlazorUI/Components/Inputs/Dropdown/BitDropdown.scss

```
@@ -213,29 +275,65 @@
         display: flex;
         flex-wrap: wrap;
         gap: spacing(0.5);
-        min-height: spacing(4);
-        padding: spacing(1) spacing(3.5) spacing(1) spacing(1);
+        min-height: var(--bit-drp-h);
+        // Logical, not physical: the room left at the end is for the icons of the trigger, which sit
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)